lib/worker/process.js

```diff
@@ -1,20 +1,105 @@
 const worker = require('worker_threads');
+const child_process = require('child_process');
 const EventEmitter = require('events');
+const Transport = require('./transport');
+const path = require('path');
+const os = require('os');
+const net = require('net');
+const fs = require('fs');
+
+const EXIT_WAIT_TIME_MS = 200;
+
+const execArgsProcessor = {
+    debugOptionPattern: /^(--inspect(?:-brk)?|--debug|--debug-(brk|port))(=\d+)?$/,
+    offset: 1,
+
+    map(execArgv) {
+        let debugPort = 0;
+        return execArgv.map(option => {
+            const debugOption = option.match(this.debugOptionPattern);
+            if (!debugOption) return option;
+            if (debugPort === 0) debugPort = process.debugPort + this.offset++;
+            return debugOption[1] + '=' + debugPort;
+        });
+    }
+};
+
+function createChannel(workerId, onConnect) {
```

Comment:
That's adaptation from:
https://github.com/mcollina/autocannon/pull/145/files
because of
https://github.com/nodejs/node/issues/14046

To make Windows work.